### PR TITLE
Theme the rename dashboard's "New name:" TextBlock

### LINKED ISSUE #6721: Inline Rename Dashboard "New name" text isn't themed (nearly unreadable in the Dark Theme)
image

### PR DESCRIPTION
**Pull request by dpoeschl**
Fixes #6721

### CODE REVIEW

**Comment by dpoeschl** [comment]
Tagging for review: @dotnet/roslyn-ide

File: src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml [hunk 1]
```diff
@@ -31,6 +31,10 @@
                 <Setter Property="Foreground" Value="{DynamicResource {x:Static utilities:CodeAnalysisColors.CheckBoxTextBrushKey}}"/>
             </Style>
 
+            <Style TargetType="TextBlock">
+                <Setter Property="Foreground" Value="{StaticResource ForegroundText}"/>
```
Comment (Pilchie) [inline src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml]: `StaticResource` is sufficient?  It doesn't need `DynamicResource` to reflect changes?

Comment (Pilchie) [inline src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml]: Ah, I see now that this is what everything else *already* did, you just extracted it to the style to make it less error-prone in the future.

**Comment by Pilchie** [comment]
:+1:

**Comment by balajikris** [comment]
:+1:

File: src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml (hunk 1 above)
Comment (jasonmalinowski) [inline src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml]: And it looks like this static resource points to the DynamicResource that is the actual thing changing.

**Comment by jasonmalinowski** [comment]
:+1:

**Pull request merged**